### ISSUE TITLE
Inconsistent training and validation PRSs

### ISSUE THREAD

**Issue by JuRaGa** [choishingwan/PRSice#179]
[PRSice_EST_IMG_Train2_50.log](https://github.com/choishingwan/PRSice/files/4107813/PRSice_EST_IMG_Train2_50.log)
[PRSice_EST_IMG_Test2_50_38.log](https://github.com/choishingwan/PRSice/files/4107819/PRSice_EST_IMG_Test2_50_38.log)

Hi,
I am deriving the optimal cut-off from a training set of individuals, and using this optimal cut-off to derive a PRS in an independent set of individuals.

Despite the prevalence of events in both training and validation sets is very similar and not statistically significantly different, I am finding striking differences in the association results. The PRS using the optimal cut-off in the training set is significantly associated (P = 0.001), but the association is flat in the validation set (P = 0.52). Trying to understand this a bit better, I have compared the distribution of both PRSs (output from PRSice "*.best") and they are significantly different too. 
How can this happen if I am using the same cut-off, and the weighting is supposed to be the same (effect sizes from the GWAS summary statistics)? These differences make me think the lack of association in the validation cohort is not physiological, but a methodological issue. 

Would you agree with this? I would appreciate your advice on this.

Thank you,

**Comment by choishingwan** [comment]
Sorry, was on leave for the past few weeks. 

I noticed that both your data set has a number of samples with invalid covariates. Does that change your sample prevalence after considering the covariate missingness? That can be one reason that leads to the drastic difference.

In addition, your p-value isn't too significant (P = 0.001 is only marginal). As PRS is affected by multiple testing and overfitting, it is not too surprising to see P-value of 0.001 not being replicate in new data set. You can test if the empirical p-value is low enough by using --perm 10000 (10000 permutations)

**Comment by JuRaGa** [comment]
Hi,

Yes, I noticed it as well and I repeated the analysis, randomly 
splitting after removing missing data. Results are still the same...

I agree the R2 is not high, so maybe the reason is probably over-fitting 
in the training set... My empirical P-value is 0.190809 (with 1000 
permutations). What does this mean exactly?

I guess I will conclude there is no association between the PRS and the 
outcome.

Thanks for your help!

**Comment by choishingwan** [comment]
The empirical p-value is calculated using permutation. An empirical p-value
of less than 0.05 (when you performed more than 10000 permutation), suggest
that your association is less likely to be affected by overfitting and
multiple testing. As you get an empirical p-value of 0.19, it is likely
that the association you observed is due to overfitting / multiple testing.
Considering that you were unable to replicate it in an independent study,
it is likely overfitted.

Sam

**Comment by JuRaGa** [comment]
Hi Sam,

This is very clear now, thank you. I will then proceed with the 
conclusions :)

Julia
